packages/appstore/src/components/compare-account/compare-account.tsx

```diff
@@ -11,11 +13,18 @@ type TCompareAccount = {
 
 const CompareAccount = ({ accounts_sub_text, is_desktop }: TCompareAccount) => {
```

Comment:
don't we need observer we need here ? since we are using useStore ?

Reply:
Okay we can add it